package.ps1

```diff
@@ -15,55 +15,72 @@
 

```

Comment:
I'm struggling to even get this script running. Any suggestions on how to improve greatly appreciated.

Reply:
Why use a custom ps1 to package when we can use nuget packaging out of the box?

Reply:
This script is not about preparing nuget, it's about preparing stuff for apache release.